broadinstitute/pearl
# participant profiles populating.  Refactored services into packages

TO TEST:
1. reset DB
2. Run populateCliApp with args "portal" "portals/ourhealth/portal.json"
3. confirm a Jonas Salk gets populated to your DB
4. edit `portals/ourhealth/participants/jsalk.json` to change some values
5. rerun the PopulateCliApp with the same arguments. 
6. Confirm updated data appears
